commit 47826cdbec2548cd1d25acf4cfaf908ae88f3325
Author: Andreas Krebbel <krebbel@linux.vnet.ibm.com>
Date:   Fri Jul 21 10:54:06 2017 +0200

    S/390: Support z14 as CPU name.
    
    With IBM z14 officially announced I can add z14 as CPU name.
    
    No regressions with that patch on s390x.
    
    gas/ChangeLog:
    
    2017-07-21  Andreas Krebbel  <krebbel@linux.vnet.ibm.com>
    
            * config/tc-s390.c (s390_parse_cpu): Add z14 as alternate CPU
            name.
            * doc/as.texinfo: Add z14 to CPU string list.
            * doc/c-s390.texi: Likewise.
    
    opcodes/ChangeLog:
    
    2017-07-21  Andreas Krebbel  <krebbel@linux.vnet.ibm.com>
    
            * s390-mkopc.c (main): Enable z14 as CPU string in the opcode
            table.

### a/opcodes/ChangeLog
### b/opcodes/ChangeLog
## -1,3 +1,8 @@
+2017-07-21  Andreas Krebbel  <krebbel@linux.vnet.ibm.com>
+
+	* s390-mkopc.c (main): Enable z14 as CPU string in the opcode
+	table.
+
 2017-07-20  Nick Clifton  <nickc@redhat.com>
 
 	* po/de.po: Updated German translation.
--- a/opcodes/s390-mkopc.c
+++ b/opcodes/s390-mkopc.c
@@ -374,7 +374,8 @@ main (void)
       else if (strcmp (cpu_string, "z13") == 0
 	       || strcmp (cpu_string, "arch11") == 0)
 	min_cpu = S390_OPCODE_Z13;
-      else if (strcmp (cpu_string, "arch12") == 0)
+      else if (strcmp (cpu_string, "z14") == 0
+	       || strcmp (cpu_string, "arch12") == 0)
 	min_cpu = S390_OPCODE_ARCH12;
       else {
 	fprintf (stderr, "Couldn't parse cpu string %s\n", cpu_string);
